benchmarks/module/module_benchmark_test.py

```diff
@@ -5,14 +5,16 @@
 from typing import List
 
 # third party
-from conv_model import run_conv_model
+from conv_model import run_inference_conv_model
+from conv_model import set_up_model
 
 
-def test_run_conv_model(benchmark, get_clients: Callable[[int], List[Any]]):
-    """Benchmark simple convolutional network.
+def test_run_inference_conv_model(benchmark, get_clients: Callable[[int], List[Any]]):
+    """Benchmark inference on simple convolutional network.
 
     Arguments:
         benchmark: Fixture that benchmarks any function passed
         get_clients: Fixture that returns a list of clients
     """
-    benchmark(run_conv_model, get_clients)
+    expected, mpc_model, x_mpc = set_up_model(get_clients)
+    benchmark(run_inference_conv_model, expected, mpc_model, x_mpc)
```

Comment:
Q: @aanurraj you said that it might be pretty hard to have 2 plots on the same graph, right?  (I was thinking if we can place the torch convolution with our MPC variant convolution)

Reply:
we can try that, i can check it on my test repo for benchmark action :D